cmd/ddev/cmd/composer-create.go

```diff
@@ -119,38 +127,33 @@ project root will be deleted when creating a project.`,
 			composerCmd = append(composerCmd, "--no-interaction")
 		}
 
+		if preferDistArg {
+			composerCmd = append(composerCmd, "--prefer-dist")
+		}
+
 		composerCmdString := strings.TrimSpace(strings.Join(composerCmd, " "))
 		output.UserOut.Printf("Executing composer command: %s\n", composerCmdString)
 		stdout, _, err := app.Exec(&ddevapp.ExecOpts{
 			Service: "web",
 			Cmd:     composerCmd,
 		})
 		if err != nil {
-			util.Failed("Failed to execute create-project command")
+			util.Failed("Failed to create project")
 		}
 
 		if len(stdout) > 0 {
 			fmt.Println(strings.TrimSpace(stdout))
 		}
 
 		output.UserOut.Printf("Moving installation to project root")
-		bashCmdString := fmt.Sprintf("if [ -d %s ]; then mv %s /var/www/html/; fi", installDir, path.Join(installDir, "*"))
+		bashCmdString := fmt.Sprintf("if [ -d %s ]; then mv %s /var/www/html/; fi", containerInstallPath, path.Join(containerInstallPath, "*"))
```

Comment:
Since this mv was one of the first and most important failures we saw, could we do it in golang on the host as well?

Reply:
I was thinking it could be done the way you did the initial directory cleanup: Walk the members of the subdirectory, and os.Rename() each.

Reply:
This has been updated to walk the tmp dir and rename its files, good suggestion!

Reply:
Thanks, I think we'll all like that better in the end.  We'll want to move this to fileutil a bit later, but we don't have to do it for this release.

You can't rename a dir to move it huh? Yuckie Shucks.